src/applications/caregivers/tests/e2e/1010cg-facilities-api.cypress.spec.js

```diff
@@ -0,0 +1,30 @@
+import path from 'path';
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+import featureTogglesFacilitiesApi from './fixtures/mocks/feature-toggles-facilities-api.json';
+import formConfig from '../../config/form';
+import manifest from '../../manifest.json';
+
+import { setupPerTest, pageHooks } from './utils/helpers';
+
+export const setupPerTestWithFacilitiesFeatureToggle = () => {
+  setupPerTest();
+  cy.intercept('GET', '/v0/feature_toggles?*', featureTogglesFacilitiesApi);
+};
+
+const testConfig = createTestConfig(
+  {
+    dataPrefix: 'data',
+    dataSets: ['requiredOnly'],
```

Comment:
I just had it run against the requiredOnly specs with the flipper turned on. We'll eventually delete this file when this is the default behavior.

Reply:
👍 For temporary flags, I've found this pattern of "copy the whole test file with the flag flipped, then delete the old one when it's not needed" to be really effective for cutting down on cleanup effort.